### ISSUE TITLE
Need more time to review changes?

### ISSUE THREAD

**Issue by kaiyuan01** [githubteacher/github-for-developers-sept-2015#373]
Compared with other VCS such as SVN, users may need more time to review/commit to master if many developers work on one file without knowing others working on it..

**Comment by jaw6** [comment]
@kaiyuan01 sorry, I'm not sure if this is a question or feedback -- are you asking us to slow down in class?

**Comment by kaiyuan01** [comment]
Sorry for the confusion. This is a general question when user uses git instead of using other VCS. As a result, will they spend more time on reviewing codes and merging changes?

**Comment by jaw6** [comment]
@kaiyuan01 thanks for clarifying: I switched from SVN to Git a long time ago, so things may be different now, but I find I spent much less time reviewing changes with Git. Branches (and merging) in SVN was typically pretty challenging, but I find it's rarely an issue in Git. That said, it's probably true that you'll need more review if your team works on the same file simultaneously than you would if you each worked on separate files. That's probably more of a team/planning issue than an issue with version control, per se.
